krod/core/vector_store.py

```
@@ -49,45 +84,105 @@ def add_document(self, text: str, metadata: Dict[str, Any] = None) -> str:
+            embeddings = self.embedding_model.encode(
+                texts,
+                show_progress_bar=len(texts) > 10,
+                convert_to_numpy=True
+            )
+            
+            # Create points for Qdrant
+            points = []
+            for i, (text, metadata, embedding) in enumerate(zip(texts, metadatas, embeddings)):
+                # Create a unique Qdrant point ID
+                point_id = i + int(uuid4().int % 10000000)  # Ensure unique IDs
+                
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)